### PR TITLE
All SetRespawnTime() calls changed to :setRespawnTime() binding calls

### PR DESCRIPTION
**Pull request by wrenffxi**
Can now remove the redundant SetRespawnTime() function in zone.lua

### CODE REVIEW

**Comment by teschnei** [comment]
If SetRespawnTime global binding is totally unused now, can you also remove
it?

I'd like to remove all of the global calls that just work on an ID with
Getmob this (if needed) and an entity function binding eventually

File: scripts/globals/zone.lua [hunk 1]
```diff
@@ -89,19 +89,3 @@ function SetExplorerMoogles(moogle)
         end
     end
 end;
-
------------------------------------
--- SetRespawnTime
-----------------------------------
-
-function SetRespawnTime(id, minTime, maxTime)
-    -- This function is redundant should place the mob:setRespawnTime() and UpdateNMSpawnPoint back in the individual zones.
```
Comment (TeoTwawki) [inline scripts/globals/zone.lua]: ❤️

**Comment by wrenffxi** [comment]
The removal of global binding is included in the PR.  Also, resolved the merge conflicts.

**Comment by teschnei** [comment]
Oh, it's just a global function, not a binding (was looking for a cpp
change). That's all good then

**Pull request merged**